app/design/frontend/base/default/layout/payone/core.xml

```diff
@@ -48,11 +48,11 @@
             <action method="setTemplate"><template>page/1column.phtml</template></action>
         </reference>
         <reference name="content">
-            <block type="payone_core/paypal_express_review" name="paypal.express.review" template="payone/core/paypal/express/review.phtml">
-                <block type="payone_core/paypal_express_review_billing" name="express.review.billing" as="billing" template="payone/core/paypal/express/review/address.phtml"/>
-                <block type="payone_core/paypal_express_review_shipping" name="express.review.shipping" as="shipping" template="payone/core/paypal/express/review/address.phtml"/>
-                <block type="payone_core/paypal_express_review" name="express.review.shipping.method" as="shipping_method" template="payone/core/paypal/express/review/shipping/method.phtml"/>
-                <block type="payone_core/paypal_express_review_details" name="paypal.express.review.details" as="details" template="payone/core/paypal/express/review/details.phtml">
+            <block type="payone_core/paypal_express_review" name="paypal.express.review" template="paypal/express/review.phtml">
+                <block type="payone_core/paypal_express_review_billing" name="express.review.billing" as="billing" template="paypal/express/review/address.phtml"/>
```

Comment:
there is one in app/design/frontend/rwd/default/template/paypal/express/review/address.phtml

Is payone plugin supposed to copy it into app/design/frontend/base ... ?

Reply:
No. Then you need to add a PAYONE layout file under `rwd/default`, which does this change. Otherwise, there will be issues with any theme, which does not inherit from `rwd/default`.

Reply:
Hi Simon,

Sorry, I don't get totally the point with the layout into 'rwd' path. 
What change should this file contain ?

Reply:
The changes including the template `paypal/express/review/address.phtml` are only relevant for the `rwd` theme. Hence, you need to create a new PAYONE layout file under `rwd/default`, which is basically the same as the one in `base/default`, but includes the necessary `rwd` changes. I am not really keen on sponsoring the work for PAYONE here, but if it really helps, I can create a PR...

Reply:
Thanks for explanation. I got the idea now. Sometimes I forget that payone core code is not magento core code, and hence is not immutable, and can be moved to more appropriate places :)
I'll include the modifications in this PR.